web/src/typing.js

```diff
@@ -129,4 +181,22 @@ export function initialize() {
             page_params.server_typing_stopped_wait_period_milliseconds,
         );
     });
+
+    $("body").on("click", ".message_edit_save, .message_edit_cancel", () => {
+        typing_status.update(
+            worker,
+            null,
+            page_params.server_typing_started_wait_period_milliseconds,
+            page_params.server_typing_stopped_wait_period_milliseconds,
+        );
+    });
```

Comment:
Please don't register extra click handlers like this. This should be done via the `message_edit` module handlers for the save/cancel operations, so that these calls can be thoughtfully sequenced with the rest of that code path and its error handling code. (E.g. if a click is not actually processed by that code, it should not issue these events).

Reply:
The reason I registered this click handler here is the `worker` object which is needed for the call back of the click event.